packages/react-devtools-extensions/src/background.js

```diff
@@ -179,26 +179,50 @@ chrome.runtime.onMessage.addListener((request, sender) => {
     if (request.hasDetectedReact) {
       setIconAndPopup(request.reactBuildType, id);
     } else {
+      const devtools = ports[id]?.devtools;
       switch (request.payload?.type) {
         case 'fetch-file-with-cache-complete':
         case 'fetch-file-with-cache-error':
           // Forward the result of fetch-in-page requests back to the extension.
-          const devtools = ports[id]?.devtools;
-          if (devtools) {
-            devtools.postMessage(request);
-          }
+          devtools?.postMessage(request);
+          break;
+        // This is sent from the backend manager running on a page
+        case 'react-devtools-required-backends':
+          const backendsToDownload = [];
+          request.payload.versions.forEach(version => {
+            if (EXTENSION_CONTAINED_VERSIONS.includes(version)) {
+              if (!IS_FIREFOX) {
+                // TODO: add equivalent logic for Firefox is in prepareInjection.js
+                chrome.scripting.executeScript({
+                  target: {tabId: id},
+                  files: [`/build/react_devtools_backend_${version}.js`],
```

Comment:
How will we keep this format of the file names? 

I am not sure if this is possible, but maybe we should add an extra check here that file `/build/react_devtools_backend_${version}.js` is present, log error otherwise

Reply:
Good point. We can try-catch and log. Our error logging only works on the internal build though.

Reply:
> How will we keep this format of the file names?

For now there is only one file. In the future I will create an automated script to take care of it.

Reply:
Actually, it is currently not possible to establish the connection to the logger in this file :(